### PR TITLE
[W6.4h][W11-B2]Oh Han Yi

### PR DESCRIPTION
**Pull request by slothhy**
Add new inteface Printable to print objects

### CODE REVIEW

File: src/seedu/addressbook/data/person/Printable.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package seedu.addressbook.data.person;
+
+/**
+ * An interface that produces a printable string representation of an object.
+ */
+public interface Printable {
+    String getPrintableString();
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Printable.java:7]: can tell what this method is supposed to do.

File: src/seedu/addressbook/ui/Formatter.java [hunk 2]
```diff
@@ -34,6 +35,17 @@ public String format(String... messages) {
         return sb.toString();
     }
 
+    /** Returns a concatenated string of each object
+     *
+     */
+    public static String getPrintableString(Printable... printables) {
```
Comment (okkhoy) [inline src/seedu/addressbook/ui/Formatter.java:41]: What would be a better place to have this method?

File: test/java/seedu/addressbook/data/PrintableTest.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+                new Address("123 Clementi", true),
+                new UniqueTagList(new Tag("ah"), new Tag("happy")));
+        ahmao = new Person(new Name("Mao"),
+                new Phone("321", false),
+                new Email("mao@gmail.com", false),
+                new Address("321 Clementi", false),
+                new UniqueTagList(new Tag("ah"), new Tag("sad")));
+    }
+
+    @Test
```
Comment (okkhoy) [inline test/java/seedu/addressbook/data/PrintableTest.java:33]: :+1: for updating the tests as well.

**Comment by okkhoy** [comment]
@slothhy 
Some comments added. Please ack & close the PR after reading comments.

File: src/seedu/addressbook/data/person/Printable.java (hunk 1 above)
Comment (slothhy) [reply src/seedu/addressbook/data/person/Printable.java:7]: Does this mean I should comment on the method instead?

**Comment by slothhy** [comment]
Acknowledged!

File: src/seedu/addressbook/ui/Formatter.java (hunk 2 above)
Comment (slothhy) [reply src/seedu/addressbook/ui/Formatter.java:41]: Should I implement this method at ReadOnlyPerson.java instead?

**Pull request closed without merging**